chore[docs]: Resolve typos in Cassandra instructions (#3266)
openebs/openebs: k8s/demo/cassandra/README.md

1. Install OpenEBS
2. Select OpenEBS storage engine
3. Configure OpenEBS LocalPV StorageClass
4. Install Kudo operator
5. Install Kudo based Cassandra
6. Verify Cassandra is up and running
7. Testing Cassandra performance on OpenEBS

### Install OpenEBS

> If OpenEBS is not installed in your K8s cluster, this can be done from [here](https://docs.openebs.io/docs/next/overview.html). If OpenEBS is already installed, go to next step.

Reviewer: ```suggestion
If OpenEBS is not installed in your K8s cluster, this can be done from [here](https://docs.openebs.io/docs/next/overview.html). If OpenEBS is already installed, go to the next step.
```

### Select OpenEBS storage engine

> A storage engine is the data plane component of the IO path of a Persistent Volume. In CAS architecture, users can choose different data planes for different application workloads based on a configuration policy. OpenEBS provides different types of storage engines and choose the right engine that suit for your type of application requirements and storage available on your Kubernetes nodes. More information can be read from [here](https://docs.openebs.io/docs/next/overview.html#openebs-storage-engines).

Reviewer: ```suggestion
A storage engine is the data plane component of the IO path of a Persistent Volume. In CAS architecture, users can choose different data planes for different application workloads based on a configuration policy. OpenEBS provides different types of storage engines and chooses the right engine that suits your type of application requirements and storage available on your Kubernetes nodes. More information can be read from [here](https://docs.openebs.io/docs/next/overview.html#openebs-storage-engines).
```

  ```
- Change the permission
  ```
  $ chmod +x kubectl-kudo
  $ sudo mv kubectl-kudo /usr/local/bin/kubectl-kudo
  ```
- Install Cert-manager

  Before installing the KUDO operator, the cert-manager must be already installed in your cluster. If not, install the cert-manager. The instruction can be found from [here](https://cert-manager.io/docs/installation/kubernetes/#installing-with-regular-manifests). Since our K8s version is v1.16.0, we have installed cert-manager using the following command.
  ```
  $ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.15.1/cert-manager.yaml
  ```

> - Install Kudo operator using specified version. In the following command, selected version is v0.14.0.

Reviewer: ```suggestion
- Install Kudo operator using a specified version. In the following command, the selected version is v0.14.0.
```
